src/main/java/net/pms/encoders/FFMpegVideo.java

```diff
@@ -954,6 +954,10 @@ public synchronized ProcessWrapper launchTranscode(
 		// Map the proper audio stream when there are multiple audio streams.
 		// For video the FFMpeg automatically chooses the stream with the highest resolution.
 		if (media.getAudioTracksList().size() > 1) {
+			// Set the video stream
+			cmdList.add("-map");
+			cmdList.add("0:v"); //TODO find a way how to automatically select propper stream when media includes multiple video streams
```

Comment:
@SubJunk maybe    `0:V`    would be better because in the FFMpeg Documentation is written that ’v’ matches all video streams, ’V’ only matches video streams which are not attached pictures, video thumbnails or cover arts.